src/app/mat-select-search/mat-select-search.component.html

```diff
@@ -6,6 +6,7 @@
       #innerSelectSearch
       class="mat-select-search-inner mat-typography mat-datepicker-content mat-tab-header"
       [ngClass]="{'mat-select-search-inner-multiple': matSelect.multiple}">
+  <mat-checkbox *ngIf="matSelect.multiple" color="primary" class="mat-select-all-checkbox" [formControl]="selectAllCheckbox"></mat-checkbox>
```

Comment:
should be configurable with an input
```suggestion
  <mat-checkbox *ngIf="_isToggleAllCheckboxVisible()" color="primary" class="mat-select-all-checkbox" [formControl]="selectAllCheckbox"></mat-checkbox>
```